### DISCUSSION TITLE
ArchLinux CI build

### DISCUSSION THREAD

**Discussion by eirnym** [openzfs/zfs#14390] [General]
Hi,

I'd like to ask if adding ArchLinux CI with fixed or the latest kernel is worth a while to create for this project? 

mostly asking @behlendorf and @tonyhutter but others are welcome to share their opinion as well. 

I'm aware, that building ArchLinux CI is quite tricky as this distribution is a rolling one, but kernel may be fixed to "latest xx.xx"

**Comment by gmelikov** [comment]
If you know a way to run it via Github actions - it may be doable, it's the preferred way now.

**Comment by eirnym** [comment]
There's a few actions out there to build Arch Linux packages, like [build-arch-package](https://github.com/marketplace/actions/build-arch-package)

**Reply by gmelikov** [reply]
Best case is to have honest Arch Linux kernel, otherwise we'll test Ubuntu's kernel (if it even starts in container, I'm not sure somebody did that for zfs-tests).

**Comment by rincebrain** [comment]
There used to be a Linux git tip CI target, it appears to be disabled right now. Probably worth asking whoever did it why before adding a new one.
